src/js/api/set-config.js

```diff
@@ -6,57 +6,41 @@ const supportedFields = [
     'mute',
     'autostart'
 ];
-let model;
 
-function setVolume(mute, volume) {
-    const muteSet = !_.isUndefined(mute);
-    const volumeSet = !_.isUndefined(volume);
-
-    if (volumeSet) {
-        model.setVolume(volume);
-        if (!muteSet) {
-            model.setMute(volume === 0);
-        }
-    }
-
-    if (muteSet) {
-        model.setMute(mute);
-    }
```

Comment:
`forEach` ensures order so `setVolume` will be called before `setMute` if both volume and mute are set. This enforces the behavior where you can set the volume and mute the player at the same time.